docs/pipelines/process/service-containers.md

```diff
@@ -85,6 +85,8 @@ resources:
     image: nginx
     ports:
     - 8080:80
+    env:
+      NGINX_PORT=80
```

Comment:
This should be `NGINX_PORT: 80`, using a `:` for yaml mapping instead of `=`

Reply:
THANK YOU! I went around on this so many time I must have copied the wrong version into these docs. 😊 I've corrected this now.